app/controllers/api/types/fabrics_controller.rb

```diff
@@ -3,8 +3,9 @@ module Types
     class FabricsController < ApiController
       def index
         chosen_fabrics = Fabric.all.select { |i| i.category.downcase == params[:type_id] }
+        serializer = ApiFabricsSerializer.new
```

Comment:
Same as in the `categories/items_controller.rb`. This should be `Types::FabricsSerializer`